docs/Contributing/API-for-contributors.md

```diff
@@ -3184,6 +3184,67 @@ Notifies the server about an agent error, resulting in two outcomes:
 
 `Status: 500`
 
+### Get the status of a device in the setup experience
+
+`GET /api/v1/fleet/device/:token/setup_experience/status`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| token | string | path | The device's authentication token. |
+
+##### Example
+
+`POST /api/v1/fleet/device/8b49859b-1ffa-483d-ad27-85b30aa3c55f/setup_experience/status`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "configuration_profiles": [
+    {
+      "profile_uuid": "39f6cbbc-fe7b-4adc-b7a9-542d1af89c63",
+      "name": "macOS restrictions",
+      "status": "acknowledged"
+    },
+    {
+      "profile_uuid": "42bcf55b-f374-46b0-b484-5ffffafaf3f6",
+      "name": "Certificates profile",
+      "status": "pending"
+    },
+  ],
+  "bootstrap_package": {
+      "name": "mac-bootstrap-package.pkg",
+      "status": "acknowledged"
+  },
```

Comment:
@marko-lisica There's one more thing that we need to wait on, it's the optional AccountConfiguration command that is sent during DEP enrollment along with installing fleetd and the bootstrap package. We currently wait for that command to be acknowledged in the worker process, so I think the new flow should also wait for it before proceeding to software installs. I'll add it to the response payload in that form, let me know if you want to tweak it:

```
"account_configuration": {
  "command_uuid": "...",
  "status": "acknowledged"
},
```

It will only be present if there is one such command (same as for bootstrap package).

Reply:
Feel free to add that to the docs. I will remove this after we implement it, since that will be orbit-authenticated endpoint.

Reply:
Added, marking as resolved.